java/runtime/src/main/java/io/ray/runtime/gcs/GlobalStateAccessor.java

```diff
@@ -78,6 +79,23 @@ private boolean connect() {
     }
   }
 
+  public byte[] getPlacementGroupInfo(PlacementGroupId placementGroupId) {
+    synchronized (GlobalStateAccessor.class) {
+      Preconditions.checkNotNull(placementGroupId,
+          "PlacementGroupId can't be null when get Placement Group info.");
+      return nativeGetPlacementGroupInfo(globalStateAccessorNativePointer,
+        placementGroupId.getBytes());
+    }
+  }
+
+  public List<byte[]> getAllPlacementGroupInfo() {
+    synchronized (GlobalStateAccessor.class) {
+      Preconditions.checkState(globalStateAccessorNativePointer != 0,
+          "Get all Placement Group info when global state accessor have been destroyed.");
```

Comment:
Please extract this check to a method and apply it to all methods.

Reply:
updated.